tools/travis-upload-to-s3.sh

```diff
@@ -3,47 +3,10 @@
 source tools/travis-helpers.sh
 
 set -euo pipefail
-IFS=$'\n\t'
 
 CT_REPORTS=$(ct_reports_dir)
 
 echo "Uploading test results to s3"
 echo $(s3_url ${CT_REPORTS})
-
-mkdir -p ${CT_REPORTS}/small
-mkdir -p ${CT_REPORTS}/big
-
-if [ -d _build/test/logs ]; then
-	cp -Rp _build/test/logs/* ${CT_REPORTS}/small
-fi
-
-CT_REPORT=test.disabled/ejabberd_tests/ct_report
-
-if [ -d ${CT_REPORT} ] && [ "$(ls -A ${CT_REPORT})" ];  then
-	cp -Rp ${CT_REPORT}/* ${CT_REPORTS}/big
-fi
-
-cat > ${CT_REPORTS}/index.html << EOL
-<html>
-  <head></head>
-  <body>
-    <p><a href="small/index.html">Small tests (test/)</a></p>
-    <p><a href="big/index.html">Big tests (test.disabled/ejabberd_tests)</a></p>
-  </body>
-</html>
-EOL
-
-now=`date +'%Y-%m-%d_%H.%M.%S'`
-LOG_DIR_ROOT=${CT_REPORTS}/logs/${now}
-for dev_node_logs_path in `find _build -name log -type d`; do
-	dev_node=$(basename $(dirname $(dirname $(dirname ${dev_node_logs_path}))))
-	LOG_DIR=${LOG_DIR_ROOT}/${dev_node}/log
-	mkdir -p ${LOG_DIR}
-	cp ${dev_node_logs_path}/* ${LOG_DIR}
-done
-
-cp *.log ${LOG_DIR_ROOT}
-cp test.disabled/ejabberd_tests/*.log ${LOG_DIR_ROOT}
-
-aws s3 sync --quiet ${CT_REPORTS} s3://mongooseim-ct-results/${CT_REPORTS}
-
+sudo pip install awscli --ignore-installed six
```

Comment:
comment about what is six and why we ignore it.

Reply:
I don't know; I moved this line from .travis.yml because it's `awscli` is only relevant for this script.

Reply:
Six is a Python 2 and 3 compatibility library.
https://pypi.python.org/pypi/six

ignore-installed flag forces pip to install new version of six, even if another version of the packet is already installed globally.